packages/filecoin-actor-utils/data/actor-codes.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Hey @navFooh the latest `react-components` has some updated logic here that includes support for `wallaby`. This looks halfway there - i think there should be an `wallaby` key here that points to an empty object?

Reply:
When we have a Lotus API URL for Wallaby, we can add it here and it will be automatically generated:
https://github.com/glifio/filecoin-descriptors/blob/primary/filecoin-descriptors.go

We can do something hacky to add an empty key to the generated object, but without any data won't that be a useless exercise?

Reply:
`'https://wallaby.dev.node.glif.io/archive/lotus/rpc/v0'`